src/System.Private.ServiceModel/tests/Common/Infrastructure/src/ConditionalWcfTest.cs

```diff
@@ -153,10 +153,8 @@ public static bool SPN_Available()
         // Returns 'true' if the server is configured to accept client certificates.
         public static bool Server_Accepts_Certificates()
         {
-            // Temporarily use the simple heuristic that if we are running the services locally, it does.
-            // Refactor this after integration to address https://github.com/dotnet/wcf/issues/1024 
-            return GetConditionValue(nameof(Server_Accepts_Certificates),
-                                     Server_Is_LocalHost);
+            //Both IIS hosted and self hosted servers accept server certificate now
+            return true;
```

Comment:
I know you aren't ready for review yet, but if this is always true, maybe we should remove this [ConditionalFact] detector and remove the [ConditionalFact(nameof(Server_Accepts_Certificate)] from all tests.  What do you think?  No change needed to the PR -- just a question.